h2o-r/h2o-package/R/kvstore.R

```diff
@@ -312,27 +313,26 @@ h2o.download_mojo <- function(model, path=getwd(), get_genmodel_jar=FALSE) {
     stop(paste0("'path',",path,", to save MOJO file cannot be found."))
   }
 
+  #Get model id
   model_id <- model@model_id
-  .__curlError = FALSE
-  .__curlErrorMessage = ""
-
-  url = .h2o.calcBaseURL(h2oRestApiVersion = .h2o.__REST_API_VERSION, urlSuffix = paste0(.h2o.__MODELS,"/",model_id,"/mojo"))
-  tmp = tryCatch(getBinaryURL(url = url,
-                      useragent = R.version.string),
-               error = function(x) { .__curlError <<- TRUE; .__curlErrorMessage <<- x$message })
-  if (! .__curlError) {
-     mojo.path <- paste0(path,"/",model_id,".zip")
-     writeBin(tmp, mojo.path, useBytes = TRUE)
-  }
+
+  #Build URL for MOJO
```

Comment:
Could this code be reused between all 3 calls:

something like `saveBinaryTo("/models/mujGreatModel.java", "mujGreatModel.java)`